prow/scripts/provision-vm-and-start-kyma.sh

```diff
@@ -2,24 +2,20 @@
 
 set -o errexit
 
+readonly SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck disable=SC1090
+source "${SCRIPT_DIR}/library.sh"
+
 cleanup() {
     ARG=$?
-    echo "
-################################################################################
-# Removing instance kyma-integration-test-${RANDOM_ID}
-################################################################################
-    "
-
+    shout "Removing instance kyma-integration-test-${RANDOM_ID}"
     gcloud compute instances delete "kyma-integration-test-${RANDOM_ID}"
     exit $ARG
 }
 
-readonly SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-# shellcheck disable=SC1090
-source "${SCRIPT_DIR}/library.sh"
 authenticate
 
-RANDOM_ID=$(< /dev/urandom tr -dc 'a-z0-9' | fold -w 8 | head -n 1)
+RANDOM_ID=$(LC_CTYPE=C < /dev/urandom tr -dc 'a-z0-9' | fold -w 8 | head -n 1)
```

Comment:
Could you add a comment about how `RANDOM_ID` is calculated? It is a bit cryptic.

Reply:
Well, I didn't implement the function, but it seems to just read from `/dev/urandom`, tr takes only the output that fits the regex, and then we trim it to the first 8 characters of the sequence.
We could have the same result using `openssl rand -hex 4`

Reply:
I know you didn't implement it :) It'd be nice to have a line comment before the calculation just to understand what the random id will be like. Now also with `LC_CTYPE` set, it got more cryptic to understand the calculation.

Reply:
I see, added comment and alternative